includes/definitions/ceragon.yaml

```diff
@@ -0,0 +1,16 @@
+os: ceragon
```

Comment:
This is too generic for an os name.

I can't find much on the FA2000, is this the only product in the range? What OS / Firmware name does it have on the device?